public/lesswrong-editor/src/ckeditor5-footnote/src/insertfootnotecommand.js

```diff
@@ -63,9 +71,10 @@ export default class InsertFootnoteCommand extends Command {
 	}
 
 	/**
+	 * Returns the footnote section if it exists, or creates on if it doesn't.
```

Comment:
`s/creates on/creates one/`

(Oddly, github didn't give me the suggest UI here.)